eth/catalyst/api.go

```diff
@@ -61,6 +62,13 @@ const (
 	invalidTipsetsCap = 512
 )
 
+var (
+	// defaultTTD is the ttd set by some CL nodes in exchangeTransitionConfiguration
+	// before the merge ttd is really set.
+	// its set to 2**256 - 2**10
+	defaultTTD = new(big.Int).Sub(new(big.Int).Lsh(big.NewInt(1), 256), new(big.Int).Lsh(big.NewInt(1), 10))
```

Comment:
Can't we use `SetString("0xfffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc00", 16)`? Or do you want to avoid handling an error here?

Reply:
I think the spec specified it as 2^256 - 2^10 so I went with that, but we can use setstring too